File: app/src/main/java/com/zofers/zofers/staff/MessageHelper.java
package com.zofers.zofers.staff;

import android.content.Context;
import android.support.annotation.Nullable;
import android.widget.Toast;

public class MessageHelper {

    public static void showErrorToast(Context context, String text) {
        Toast.makeText(context, "something went wrong " + text, Toast.LENGTH_SHORT).show();
    }

    public static void showNoConnectionToast (Context context) {
        showErrorToast(context, "no connection");
    }
}

File: app/src/main/java/com/zofers/zofers/staff/MessageHelper.kt
package com.zofers.zofers.staff

import android.content.Context
import android.view.View
import android.widget.Toast
import com.google.android.material.snackbar.Snackbar


object MessageHelper {

    fun showErrorToast(context: Context?, text: String = "", duration: Int = Toast.LENGTH_SHORT) {
        Toast.makeText(context, "something went wrong $text", duration).show()
    }

    fun showNoConnectionToast(context: Context?) {
        showErrorToast(context, "no connection")
    }

    fun showSnackBar(view: View, text: String, duration: Int = Snackbar.LENGTH_SHORT) {
        Snackbar.make(view, text, Snackbar.LENGTH_SHORT).show()
    }


}
